src/main/java/com/karl/Globals.java

```diff
@@ -0,0 +1,6 @@
+package com.karl;
+
+public class Globals {
+  public static int PORT = 7777;
+  public static String HOST = "127.0.0.1";
```

Comment:
I like putting this stuff in a Constants folder.

Reply:
+1

Reply:
Done!